### ISSUE TITLE
Orchard Gallery Url is broken

### ISSUE THREAD

**Issue by JingyuShao** [OrchardCMS/Orchard#7367]
I'm using Orchard 1.9.1.0, and the current Url I've been keep using http://packages.orchardproject.net/FeedService.svc
seems to be broken, as I cannot connect to Gallery from Orchard dashboard.
Such error reported:
Error loading extensions from gallery source 'Orchard Gallery'. Unable to connect to the remote server.

**Comment by sebastienros** [comment]
Please change you Gallery Source to https://gallery.orchardproject.net/api/FeedService

**Comment by JingyuShao** [comment]
yeah it worked. Is that a replacement of the previous one? Should I update all my environment to use the one you provided?

**Comment by sebastienros** [comment]
Yes.

**Comment by sebastienros** [comment]
But you should not have to browse the gallery on a production environment.

**Comment by ancientyouthman** [comment]
Where do we change the package gallery source to  https://gallery.orchardproject.net/api/FeedService ?

**Comment by sfmskywalker** [comment]
You can update the URL via Settings -> Gallery. There you'll see a page with a gallery feed. You can either update it or delete it and create a new one, not sure which but you'll see.

**Comment by ancientyouthman** [comment]
sick thanks

**Comment by visualinterface** [comment]
Hey guys,
I literally just downloaded and installed the latest version of Orchard.  I configured multi tenancy, and setup a blog tenant.  I went in to the admin of the main tenant, deleted the existing gallery, and added the new one as specified in this thread: https://gallery.orchardproject.net/api/FeedService

However, I still see the images are missing when I hit the themes gallery, and they are still pointing to: http://orchardgallery.azurewebsites.net.

Any ideas?

Orchard newbie
